content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/package.rst

```diff
@@ -0,0 +1,219 @@
+   - :doc:`Picking method: batch <batch_transfers>`
+   - :doc:`Picking method: cluster <cluster_picking>`
+   - :doc:`Picking method: wave <wave_transfers>`
+
+First, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and activate the
+:guilabel:`Batch Transfers` feature, located in the :guilabel:`Operations` section. Doing so makes
+the :guilabel:`Package Use` field become visible on a package form.
+
+.. image:: package/enable-batch.png
+   :align: center
+   :alt: Activate the *Batch Transfers* feature in Inventory > Configuration > Settings.
+
+Then, go to the package form to configure the package intended for batch picking. To do that,
+navigate to :menuselection:`Inventory app --> Products --> Packages`.
+
+On this page, add new packages by clicking :guilabel:`New`, or select an existing package. Doing so
+opens the package form, which contains the following fields:
+
+- :guilabel:`Package Reference`: (required) name of the package
+- :guilabel:`Pack Date`: the date that the package was created
+- :guilabel:`Package Use`: choose :guilabel:`Reusable` for packages used for moving products within
+  the warehouse; :guilabel:`Disposable` for packages used to ship products to customers.
+- :guilabel:`Package Type`: used for :ref:`configuring shipping boxes to ship to the customer
+  <inventory/warehouses_storage/package-type>`.
+
+  .. note::
+     :guilabel:`Package Type` is unnecessary for configuring batch transfer packages.
+
+- :guilabel:`Shipping Weight`: used to input the weight of the package after measuring it on a
+  scale.
+- :guilabel:`Company`: specify a company to make the package available **only** at the selected
+  company. Leave the field blank if the package is available at all companies.
+
+.. image:: package/package.png
+   :align: center
+   :alt: Display package form to create a cluster pack.
+
+Add to batch
+------------
+
+.. important::
+   The following example uses packages in a :doc:`three-step delivery
+   <../../shipping_receiving/daily_operations/delivery_three_steps>` workflow.
+
+   This workflow requires the :guilabel:`Packages`, :guilabel:`Storage Locations`, and
+   :guilabel:`Multi-Step Routes` features to be enabled in :menuselection:`Inventory app -->
+   Configuration --> Settings`. On the warehouse configuration page (:menuselection:`Inventory app
+   --> Configuration --> Warehouses`), the :guilabel:`Outgoing Shipments` option must be set to
+   :guilabel:`Pack goods, send goods in output and then deliver (3 steps)`.
+
+After confirming a sales order, selecting the :guilabel:`Delivery` smart button displays the three
+tasks to complete the delivery.
+
+.. image:: package/three-step.png
+   :align: center
+   :alt: Clicking "Delivery" smart button displays the pick, pack, ship steps.
+
+To create a batch transfer, navigate to :menuselection:`Inventory app --> Operations --> Internal`
+and select the checkboxes next to the pick operations that are to be added to the batch.
```

Comment:
I'm not seeing an 'Internal' option under the Operations menu in the Inventory app (while using a 16.0 runbot)
https://drive.google.com/file/d/1nKVQIrugAi8kfLnAejCxhZrdIApG34Dk/view?usp=sharing

Reply:
Are you intending them to navigate to the Internal Transfers page, from the Internal Transfers card from the Inventory Overview page? If so, some clearer directions would be super helpful.